### ISSUE TITLE
@honeybadger-io/react does not contain a default export as described in the docs

### ISSUE THREAD

**Issue by garethlau** [honeybadger-io/honeybadger-react#276]
## What are the steps to reproduce this issue?
1. Install `@honeybadger-io/react`
2. Import the `ErrorBoundary` component. [Per the docs](https://docs.honeybadger.io/lib/javascript/integration/react/):
```jsx
import ErrorBoundary from '@honeybadger-io/react'
```

## What happens?
Produces an error:
`Attempted import error: '@honeybadger-io/react' does not contain a default export (imported as 'ErrorBoundary').`

## What were you expecting to happen?
…

## Any logs, error output, etc?
`Attempted import error: '@honeybadger-io/react' does not contain a default export (imported as 'ErrorBoundary').`
<img width="858" alt="image" src="https://user-images.githubusercontent.com/43855680/134816439-c15793a3-257c-4718-a252-c46e6aee006d.png">

## Any other comments?
This seems like just an issue of the docs being out of date. Can fix it like so:
```jsx
import { HoneybadgerErrorBoundary as ErrorBoundary } from '@honeybadger-io/react';
```

## What versions are you using?
**Package Version:**  @honeybadger-io/react": "^2.0.0

**Comment by subzero10** [comment]
Hello @garethlau,

Thanks for submitting an issue!

As you have correctly guessed, the problem you are facing is because we have just released version `2.0.0` and haven't updated the docs yet. We are going to do that asap.

The major change is that now both `Honeybadger` and `HoneybadgerErrorBoundary` (formerly `ErrorBoundary`) are exported from `@honeybadger-io/react`. Therefore this:
```
import Honeybadger from '@honeybadger-io/js'
import ErrorBoundary from '@honeybadger-io/react'
```
is now:
```
import {Honeybadger, HoneybadgerErrorBoundary} from '@honeybadger-io/react'
```

cc @joshuap

**Comment by joshuap** [comment]
I just deployed the updated documentation for this:

https://docs.honeybadger.io/lib/javascript/integration/react/#installation

Thanks for the heads up!
